packages/overlay/src/index.tsx

```diff
@@ -251,6 +263,16 @@ export const Overlay: React.FC<OverlayProps> = ({
   )
 }
 
+type OverlayVerticalAlign =
```

Comment:
This should contain only `top, center, bottom` or `start, center, end`, we don't need all these values. Right?

Reply:
Correct, left `stretch` anyway due to different behavior than `top`